arch/arm/configs/bcm2709_defconfig

```diff
@@ -1394,6 +1396,10 @@ CONFIG_SPS30_I2C=m
 CONFIG_MAX30102=m
 CONFIG_DHT11=m
 CONFIG_HDC100X=m
+CONFIG_HDC3020=m
+CONFIG_HTS221=m
+CONFIG_HTS221_I2C=m
+CONFIG_HTS221_SPI=m
```

Comment:
These two (HTS221_I2C and HTS221_SPI) are the defaults if HTS221 is enabled - drop them.